service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -356,7 +356,7 @@ fn generate_body_parser(operation: &Operation, service: &Service<'_>) -> String
                 Some(ref s) => {
                     // if there's any required shape present the body payload parser will handle it
                     // This can't be converted to `s.is_empty()`. TODO: find out why.
```

Comment:
Are these comments above still valid?

Reply:
Oh i forgot to remove it. Before rust 1.39.0 `make generate` throwed an error when this change  was made

Reply:
Ah okay got it, 👍 👍